### ISSUE TITLE
[Designer- Publish Adaptive Card Schema]: On activating 'Set due date' Narrator announces all the information present under the 'Set due date' field at once under 'Publish Adaptive Card Schema'.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7309]
https://microsoft.visualstudio.com/OS/_workitems/edit/37777384

**Comment by vagpt** [comment]
Hi @anna-dingler,

Could you please let us know update regarding this issue fixing as of now this issue is still repro.

Test Environment:

URL: https://adaptivecards.io/

OS Version: Dev( Build 26002.1000)

Browser: Edge Dev( 121.0.2277.4)

Screen reader: Narrator

Repro steps:

1. Press Ctrl+ Windows+ Enter key to turn on the Narrator.

2.Launch the above URL in edge dev browser and login with v- credentials.
3. Navigate to 'Designer' link through tab key under the header controls and activate it.
4. Navigate to the 'Set Due date' control under 'Publish Adaptive Card Schema' card and activate it.

5. Verify that on activating 'Set due date' control Narrator announces all the information present under the 'Set Due Date' edit field or not.


Actual Result:
On activating 'Set due date' Narrator announces all the information present under the 'Set due date' field at once under 'Publish Adaptive Card Schema'.

Expected Result:
On activating 'Set due date' Narrator should not announce all the information present under the 'Set due date' field at once under 'Publish Adaptive Card Schema'. It should only announce as 'Set due date' menu item expanded 1 of 2'

Observation:
Issue does not repro with NVDA.

User Experience:
Screen reader user will get impacted if Narrator start reading all the information at once due to which user will not be able to clearly understand the purpose of the control and will not be able to track the content properly.

MAS Reference Link:
[Info and Relationships - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.01)

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “,
